src/Polly.Strategies/Predicates.cs

```diff
@@ -0,0 +1,57 @@
+﻿namespace Polly;
+
+public delegate ValueTask<bool> PredicatesDelegate<T, TContext>(Outcome<T> outcome, TContext context);
```

Comment:
Best not to use the word "Delegate" in the name. Just like we don't call classes FooClass. Don't focus on the programming construct, focus on what the delegete is semantically for.

Reply:
Renamed to same term as used in docs `PredicatesCallback`